pkg/task/backup.go

```diff
@@ -179,3 +197,25 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 	}
 	return nil
 }
+
+// parseTSString port from tidb setSnapshotTS
+func parseTSString(ts string) (uint64, error) {
+	if len(ts) == 0 {
+		return 0, nil
+	}
+	if tso, err := strconv.ParseUint(ts, 10, 64); err == nil {
+		return tso, nil
+	}
+	sc := &stmtctx.StatementContext{
+		TimeZone: time.Local,
+	}
+	t, err := types.ParseTime(sc, ts, mysql.TypeTimestamp, types.MaxFsp)
+	if err != nil {
+		return 0, errors.Trace(err)
+	}
+	t1, err := t.GoTime(time.Local)
+	if err != nil {
+		return 0, errors.Trace(err)
+	}
+	return variable.GoTimeToTS(t1), nil
```

Comment:
this looks very heavyweight to support the additional timestamp format. wdyt if we only allow TSO?

Reply:
yes it's heavyweight , but I think unified the usage with tidb `set tidb_snapshot` is better, TSO is not easy for general users to get, @overvenus what's your opinion?

Reply:
LGTM.

Reply:
which looks good? 😅

Reply:
`variable.GoTimeToTS` is LGTM.